### PR TITLE
Update sent_deployableballoons.lua

### PR DESCRIPTION
**Pull request by Python1320**
Trying to create entities too early! (filter_activator_name)

### CODE REVIEW

**Comment by Divran** [comment]
what does this thing even do

**Comment by oldmud0** [comment]
```lua
ENT.Purpose         = "It Deploys Balloons."
ENT.Instructions = "Use wire."
```
The real question is, does anyone even *use* this? Should this be put in wire-extras?

**Comment by thegrb93** [comment]
It's been a wire SENT since gmod 10 lol. I've only used it for hotair balloons, but there's other creative things for it.

**Comment by Divran** [comment]
yeah this entity as a whole is pretty useless. The only thing I've used it for is `@outputs X runOnTick(1) X=!X` lol. Most servers probably get an admin mod that can block tools and then block this

**Comment by AbigailBuccaneer** [comment]
It's a very cute little entity, and I personally think it should be kept in Wiremod just for its historical significance if nothing else :relaxed:

File: lua/entities/sent_deployableballoons.lua [hunk 1]
```diff
@@ -34,13 +34,13 @@ CreateConVar('sbox_maxwire_deployers', 2)
 local DmgFilter
 
 local function CreateDamageFilter()
-	if DmgFilter then return end
+	if IsValid(DmgFilter) then return end
 	local DmgFilter = ents.Create("filter_activator_name")
```
Comment (AbigailBuccaneer) [inline lua/entities/sent_deployableballoons.lua]: You didn't write this code, but this avoid-double-initialization logic is wrong: by declaring `local DmgFilter` here, the `local DmgFilter` on line 34 never gets assigned to.

Comment (bigdogmat) [inline lua/entities/sent_deployableballoons.lua]: It shouldn't even be needed anymore. `InitPostEntity` should only fire once, where `Initialize` will fire every time the gamemode is reloaded (e.g. autorefresh).

**Comment by thegrb93** [comment]
It also makes a good confetti particle emitter if you attach a pulser to it xD

File: lua/entities/sent_deployableballoons.lua (hunk 1 above)
Comment (AbigailBuccaneer) [inline lua/entities/sent_deployableballoons.lua]: Fair enough - in that case can you remove it please?

**Comment by Python1320** [comment]
removed the extra local

**Pull request merged**